### ISSUE TITLE
Alphabetical order

### ISSUE THREAD

**Issue by hiendv** [VietFOSS/vn-foss#15]
Lists are now updated by appending latest items. They should go on forever. Alphabetical order makes it easier to discover people and projects.

**Comment by vietlq** [comment]
Thanks @hiendv. That will be done when we have a site/Google form for ppl submit :D Or we can sort before committing.

**Comment by hiendv** [comment]
I would recommend a CI for the automated fix. TravisCI for example.

**Comment by vietlq** [comment]
@hiendv  Indeed, I was thinking about it too :) We will need a simple parser/formatter to run with CI.
